### ISSUE TITLE
Draft new library descriptions

### ISSUE THREAD

**Issue by mattrosno** [carbon-design-system/carbon-platform#559]
Here are the library descriptions that will show up on library detail pages once #558 is complete. This issue is to specify library description changes.

- https://github.com/carbon-design-system/carbon-charts/blob/master/packages/angular/carbon.yml#L5
- https://github.com/carbon-design-system/carbon-charts/blob/master/packages/core/carbon.yml#L5
- https://github.com/carbon-design-system/carbon-charts/blob/master/packages/react/carbon.yml#L5
- https://github.com/carbon-design-system/carbon-charts/blob/master/packages/svelte/carbon.yml#L5
- https://github.com/carbon-design-system/carbon-charts/blob/master/packages/vue/carbon.yml#L5
- https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/packages/react/carbon.yml#L5
- https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/packages/web-components/carbon.yml#L5
- https://github.com/carbon-design-system/carbon/blob/e5d1cad837798223958d166a4f0ccfba975d3b03/packages/react/carbon.yml#L4

**Comment by mattrosno** [comment]
@aubrey-oneal the library descriptions also show up in the libraries page. Do you think we can come up with fitting library descriptions that show up both here and in the library detail page? It would be nice to have one description and not separate short-form and long-form descriptions.

Here's the guidance we have in the schema:

https://github.com/carbon-design-system/carbon-platform/blob/main/docs/resource-schemas.md#library-keys

> Library description ideally between 50-160 characters in length. Use sentence-case capitalization.

<img width="1392" alt="image" src="https://user-images.githubusercontent.com/1691245/163267347-49728393-3240-4042-a3d6-01416c452701.png">

**Comment by mattrosno** [comment]
@kingtraceyj is this part of the current sprint 17? Or is it queued up for sprint 18? Or should it be in the design backlog?

**Comment by aubrey-oneal** [comment]
Reviewers: could you please look at the airtable? 

If you like my draft, copy it into 'Final' and mark it as 'approved'. Or check out my guidelines (above) and edit your own version.

@ljcarot @mattrosno @jeanservaas @Morgana @alisonjoseph

**Comment by yumeforever** [comment]
@aubrey-oneal I followed up with David and Laura, just to make sure, but I think it looks great so far. I will get back to you once I hear from them (probably today).

**Comment by Levinson1** [comment]
@aubrey-oneal @yumeforever Can you please add a representative from IBM Products to this and any other workstreams that require IBM Products input? Laura Marshall and/or Carrie Lloyd would be best. Thanks.

As for the description, as this is for the code library, I have asked our FED Manager, Elysia Hwang for her input.

**Comment by aubrey-oneal** [comment]
Absolutely @Levinson1!

**Comment by aubrey-oneal** [comment]
The current IBM product library description I have in our airtable is: Find solutions designed for IBM's product portfolio.

(copying here because I don't think everyone can access that airtable)

**Comment by Levinson1** [comment]
From Elysia:
React-based solutions for IBM’s product portfolio

**Comment by aubrey-oneal** [comment]
Ok I'll chat with her. I'd like to find something that follows of the content guidelines (above) so that we can have a cohesive set of descriptions across all libraries.

**Comment by alisonjoseph** [comment]
The core library description all look good to me 👍

**Comment by mattrosno** [comment]
@aubrey-oneal drafts look great here!

**Comment by mattrosno** [comment]
@aubrey-oneal here are additional indexed libraries that don't show up in the core system, but could become visible in local systems.

https://next.carbondesignsystem.com/libraries/carbon-styles

https://next.carbondesignsystem.com/libraries/ibmdotcom-styles

https://next.carbondesignsystem.com/libraries/ibmdotcom-utilities

https://next.carbondesignsystem.com/libraries/ibmdotcom-services

You can always direct link to these. The styles libraries may never become visible in the libraries catalog, but I definitely expect Carbon for IBM.com local system to show those utilities and services libraries in its libraries catalog.
